frontend/src/services/AdvisorService.ts

```diff
@@ -14,3 +14,20 @@ export const getStudents = (
       Authorization: "Token " + userToken,
     },
   }).then(response => response.json());
+
+/* Service function object to find all templates given a search query
+ * @param searchQuery  the search query
+ * @param pageNumber  page number for the query
+ * @param userToken the JWT token of the user
+ */
+export const getTemplates = (
+  searchQuery: string,
+  pageNumber: number,
+  userToken: string
+) =>
+  fetch(`/api/users/students?search=${searchQuery}&page=${pageNumber}`, {
```

Comment:
Is this just temporary?

Reply:
Yeah, we just had it in as a placeholder since the API isn't set up yet, but we'll remove it for now to avoid confusion.